lib/workers/repository/update/pr/changelog/release-notes.ts

```diff
@@ -166,10 +173,13 @@ export async function getReleaseNotes(
 
 function getExactReleaseMatch(
   packageName: string,
+  depName: string,
   version: string,
   releases: ChangeLogNotes[],
 ): ChangeLogNotes | undefined {
-  const exactReleaseReg = regEx(`${packageName}[@_-]v?${version}`);
+  const exactReleaseReg = regEx(
+    `(?:^|/)(?:${packageName}|${depName})[@_-]v?${version}`,
```

Comment:
`/` included here since release tag may have a prefix (delimited by `/`) which is not part of the `packageName` (cf. [related test](https://github.com/renovatebot/renovate/blob/70cc17fc397f8e652a6ff8e85dbbd63c3fa6aa07/lib/workers/repository/update/pr/changelog/release-notes.spec.ts#L862-L911)).